package.json

```diff
@@ -17,7 +17,7 @@
     "@vitejs/plugin-react-refresh": "^1.3.5",
     "@wojtekmaj/enzyme-adapter-react-17": "^0.6.2",
     "clsx": "^1.1.1",
-    "enzyme": "^3.11.0",
```

Comment:
🔧 We'd better leave enzyme in the project.

Reply:
Enzyme added back in.